client/src/components/ItemList/Item.js

```diff
@@ -98,6 +98,10 @@ export default function Item({ item }) {
   };
 
   const me = auth.getProfile(); // me.data.username
+  if (!user) {
+    console.log('no user');
+    return <></>;
```

Comment:
This will actually not show the item if the user isn't added properly... we might not want this behavior